CI.hs

```diff
@@ -0,0 +1,89 @@
+-- Copyright (c) 2019 Digital Asset (Switzerland) GmbH and/or its affiliates. All rights reserved.
+-- SPDX-License-Identifier: Apache-2.0
+
+-- CI script to build the DA GHC fork and make a ghc-lib out of it
+
+import Control.Monad
+import System.Directory
+import System.Process.Extra
+import System.IO.Extra
+import System.Info.Extra
+import System.Exit
+import System.Time.Extra
+import Data.List.Extra
+import System.Environment
+
+main :: IO ()
+main = do
+    let cmd x = do
+        putStrLn $ "\n\n# Running: " ++ x
+        hFlush stdout
+        (t, _) <- duration $ system_ x
+        putStrLn $ "# Completed in " ++ showDuration t ++ ": " ++ x ++ "\n"
+        hFlush stdout
+    when isWindows $
+        cmd "stack exec -- pacman -S autoconf automake-wrapper make patch python tar --noconfirm"
+
+    args <- getArgs
+    let featureBranch = args !! 0
```

Comment:
Still can't help but feel we should look more closely at this but it will do to get us going.